AcademicReward/PopUps/EditShopItemPage.xaml

```diff
@@ -0,0 +1,99 @@
+                    VerticalOptions="Center"
+                    TextDecorations="Underline"/>
+            </Frame>
+        </StackLayout>
+
+        <StackLayout Margin="10">
+            <Label
+                Text="Item Name:"
+                FontSize="22"
+                Margin="0, 5"
+                FontAttributes="Bold"
+                VerticalOptions="Center"/>
+            <Frame BackgroundColor="{StaticResource Gray200}" BorderColor="{StaticResource Gray200}" Margin="0, 5">
+                <Entry x:Name="name" FontSize="20" Placeholder="Item Name..." PlaceholderColor="{StaticResource Gray400}" BackgroundColor="{StaticResource Gray200}"/>
+            </Frame>
+            <Label
+                Text="Item Description:"
+                FontSize="22"
+                Margin="0, 5"
+                FontAttributes="Bold"
+                VerticalOptions="Center"/>
+            <Frame BackgroundColor="{StaticResource Gray200}" BorderColor="{StaticResource Gray200}" Margin="0, 5">
+                <Entry x:Name="description" FontSize="20" Placeholder="Item Description..." PlaceholderColor="{StaticResource Gray400}" BackgroundColor="{StaticResource Gray200}"/>
+            </Frame>
+            <Label
+                Text="Item Cost:"
+                FontSize="22"
+                Margin="0, 5"
+                FontAttributes="Bold"
+                VerticalOptions="Center"/>
+            <Frame BackgroundColor="{StaticResource Gray200}" BorderColor="{StaticResource Gray200}" Margin="0, 5">
+                <Entry x:Name="cost" FontSize="20" Placeholder="Item Cost..." PlaceholderColor="{StaticResource Gray400}" BackgroundColor="{StaticResource Gray200}"/>
+            </Frame>
+            <Label
+                Text="Level Requirement:"
+                FontSize="22"
+                Margin="0, 5"
+                FontAttributes="Bold"
+                VerticalOptions="Center"/>
+            <Frame BackgroundColor="{StaticResource Gray200}" BorderColor="{StaticResource Gray200}" Margin="0, 5">
+                <Entry x:Name="levelRec" FontSize="20" Placeholder="Level Requirement..." PlaceholderColor="{StaticResource Gray400}" BackgroundColor="{StaticResource Gray200}"/>
+            </Frame>
+            <Label
+                Text="Group: "
+                FontSize="22"
+                Margin="0, 5"
+                FontAttributes="Bold"
+                VerticalOptions="Center"/>
+            <Frame BackgroundColor="{StaticResource Gray200}" BorderColor="{StaticResource Gray200}" Margin="0, 5">
+                <Picker Title="Select A Group..." x:Name="GroupPicker" FontSize="20">
+                    <Picker.ItemsSource>
+                        <x:Array Type="{x:Type x:String}">
+                            <x:String>Group 1</x:String>
+                            <x:String>Group 2</x:String>
+                            <x:String>Group 3</x:String>
+                            <x:String>Group 4</x:String>
```

Comment:
This static list can be removed if we are dynamically populating the list now

Reply:
I'll replace the static list with one that checks the admins groups in the next sprint, for now leaving it in as discussed